shared/github_oauth.go

```diff
@@ -106,16 +114,11 @@ func (g *githubOAuthImpl) GetGitHubUser(client *github.Client) (*github.User, er
 func NewGitHubOAuth(ctx context.Context) (GitHubOAuth, error) {
 	store := NewAppEngineDatastore(ctx, false)
 	log := GetLogger(ctx)
-	clientID, err := GetSecret(store, "github-oauth-client-id")
-	if err != nil {
-		log.Errorf("Failed to get github-oauth-client-id secret: %s", err.Error())
-		return &githubOAuthImpl{}, err
-	}
 
-	secret, err := GetSecret(store, "github-oauth-client-secret")
+	clientID, secret, err := getOAuthClientIDSecret(store)
 	if err != nil {
-		log.Errorf("Failed to get github-oauth-client-secret: %s", err.Error())
-		return &githubOAuthImpl{}, err
+		log.Errorf("Failed to get github-oauth-client-{id,secret}: %e", err)
```

Comment:
```suggestion
		log.Errorf("Failed to get github-oauth-client-{id,secret}: %s", err.Error())
```
%e is used for scientific notation?

Reply:
Ahh thanks so,ejpw I thought it was for error...